mongo/integration/change_stream_test.go

```diff
@@ -586,6 +586,23 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			assert.Equal(mt, 2, numClearedEvents, "expected two PoolCleared events, got %d", numClearedEvents)
 		})
 	})
+	mt.RunOpts("call to cursor.Next after cursor closed", mtest.NewOptions().MinServerVersion("4.0"), func(mt *mtest.T) {
+		cs, err := mt.Coll.Watch(mtest.Background, mongo.Pipeline{})
+		assert.Nil(mt, err, "Watch error: %v", err)
+		defer closeStream(cs)
+		// Generate insert events
+		generateEvents(mt, 5)
+		// Call Coll.Drop to generate drop and invalidate event
+		assert.Nil(mt, mt.Coll.Drop(mtest.Background), "expected Nil but returned %v", err)
+		// Test that all events were successful
+		for i := 0; i < 7; i++ {
+			assert.True(mt, cs.Next(mtest.Background), "expected to return True, but returned False")
+		}
+		operationType := cs.Current.Lookup("operationType").String()
+		assert.Equal(mt, operationType[1:len(operationType)-1], "invalidate", "expected invalidate event but returned %s event", operationType)
```

Comment:
In your assertion, prefer using `%q` over `%s` because that will put quotes around the string. The `invalidate` in the assertion should have quotes around it too for readability.

Reply:
done